lib/nerves_network/if_supervisor.ex

```diff
@@ -247,8 +253,12 @@ defmodule Nerves.Network.IFSupervisor do
 
     managers_v4 = ipv4_managers(settings)
     managers_v6 = ipv6_managers(settings)
+    eapol       = eapol_managers(settings)
+
+    managers =
```

Comment:
if you don't have any debug messages in between, I'd go back to previous version without the `managers` variable in between.